src/controller/controller.c

```diff
@@ -483,37 +483,62 @@ static bool controller_setup_node_connection_handler(Controller *controller) {
 
 static int controller_reset_heartbeat_timer(Controller *controller, sd_event_source **event_source);
 
+static int controller_check_node_liveness(Controller *controller, Node *node) {
```

Comment:
Since this is returning only two values, lets use a `bool` here instead.

Reply:
done